pages/searchDetail/searchDetail.js

```diff
@@ -0,0 +1,43 @@
+import testData from './test.js';
+Page({
+
+  /**
+   * 页面的初始数据
+   */
+  data: {
+    detail:{},
+    userInfo:{}
+  },
+
+  /**
+   * 生命周期函数--监听页面加载
+   */
+  onLoad: function (options) {
+    this.getData(options.fullName);
+  },
+
+//  获取数据方法
+  getData(param){
+    let that = this;
+    wx.showLoading({
+      title: '加载中',
+    })
+    wx.request({
+      url: 'https://api.github.com/repos/' + param,
```

Comment:
Would it break if the request responding with `404`?
There's a quirk in the `wx.request` API, check the DOC.